### PR TITLE
Adding a post for Autograd (and JAX)

### PR DESCRIPTION
**Pull request by tabakg**

### CODE REVIEW

File: _posts/software/2020-10-18-autograd.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+<br>
+
+We encourage you to contribute to the [research software encyclopedia](https://rseng.github.io/rse/tutorials/annotation/) and annotate the respository:
+
+<ul>
+<li><a href="{{ page.annotate_criteria }}" target="_blank">Annotate the software criteria</a></li>
+<li><a href="{{ page.annotate_taxonomy }}" target="_blank">Annotate the software taxonomy</a></li>
+</ul>
+
+otherwise, keep reading!
```
Comment (vsoch) [inline _posts/software/2020-10-18-autograd.md]:
```suggestion
```
The sentence from above was changed, so this should be removed.

File: _posts/software/2020-10-18-autograd.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+
+<a id="what-is">
```
Comment (vsoch) [inline _posts/software/2020-10-18-autograd.md:37]: Can you please add back in the small navigation that pointed to these sections? E.g., "What is Autograd," etc

File: _posts/software/2020-10-18-autograd.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+<a id="JAX">
+## But does it scale?!
+
+That's the milion dollar question these days. Luckily for us, the vision of Autograd is being carried on in the high-performance context with [JAX](https://github.com/google/jax), which utilizes XLA (Accelerated Linear Algebra), originally developed for Tensorflow. Although this is still a young library, it promises a lot by combining the best of both worlds.
+
+
+<a id="cite">
+## How do I cite it?
+
+You can cite statsmodels as follows, a pdf of the paper is available <a href="https://indico.ijclab.in2p3.fr/event/2914/contributions/6483/subcontributions/180/attachments/6060/7185/automl-short.pdf" target="_blank">here</a>:
```
Comment (vsoch) [inline _posts/software/2020-10-18-autograd.md]:
```suggestion
You can cite autograd as follows, and a pdf of the paper is available <a href="https://indico.ijclab.in2p3.fr/event/2914/contributions/6483/subcontributions/180/attachments/6060/7185/automl-short.pdf" target="_blank">here</a>:
```

File: _posts/software/2020-10-18-autograd.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+</ul>
+
+or read more about annotation [here]({{ site.baseurl }}/tutorials/annotate-your-software). You can clone the software repository to do
+bulk annotation, or annotation any repository in the <a href="https://rseng.github.io/software/" target="_blank">software database</a>,
+We want annotation to be fun, straight-forward, and easy, so we will be showcasing one repository to annotate per week.
+If you'd like to request annotation of a particular repository (or addition to the software database)
+please don't hesitate to [open an issue](https://github.com/rseng/software/issues) or even a pull request.
+
+<a id="learn-more">
+## Where can I learn more?
```
Comment (vsoch) [inline _posts/software/2020-10-18-autograd.md:139]: Learn more shouldn't be changed from the original template - the idea is to learn about the RSEPedia. These links should go up under "Getting Started"

**Review by vsoch — COMMENTED** [review]
See comments for changes requested! Also, please don't add .DS_Store files to the repo - if you want to remove them and add to the .gitignore so it doesn't happen again, that works too.

**Comment by vsoch** [comment]
You mentioned in the title this is for autograd AND jax - are these separate repos?

**Comment by tabakg** [comment]
> You mentioned in the title this is for autograd AND jax - are these separate repos?

Yes, but closely related

**Comment by tabakg** [comment]
> See comments for changes requested! Also, please don't add .DS_Store files to the repo - if you want to remove them and add to the .gitignore so it doesn't happen again, that works too.

Oh I didn't see this, I'll remove them

File: _posts/software/2020-10-18-autograd.md (hunk 4 above)
Comment (tabakg) [reply _posts/software/2020-10-18-autograd.md:139]: Changed it back

File: _posts/software/2020-10-18-autograd.md (hunk 2 above)
Comment (tabakg) [reply _posts/software/2020-10-18-autograd.md:37]: Added back

**Review by vsoch — APPROVED** [review]

**Pull request merged**